srtp/srtp.c

```diff
@@ -769,13 +769,13 @@ static inline int base_key_length(const srtp_cipher_type_t *cipher, int key_leng
   case SRTP_AES_ICM_256:
     /* The legacy modes are derived from
      * the configured key length on the policy */
-    return key_length - 14;
+    return key_length - SRTP_SALT_LEN;
```

Comment:
A case of trying to be consistent but maybe failing.
AES_ICM was a fall through case statement with computed value where as the AES_GCM where individual cases with constant value.  You had previously expressed a dislike for fall though cases so I was reluctant to change it here.
Could change the AES_GCM to also be fall through and used computed value or else change the AES_ICM to be separate cases and used constant values. Do you have a preference?

This whole function is a bit strange, and really should look at where it is called from, why is key_length even needed if we have constant defines for all cipher types. But once again this change was not meant to refactor code just use defines instead of literals.